### DISCUSSION TITLE
[BUG] Bad Gateway API error in listings /shops/<id>/listings for a very specific item

### DISCUSSION THREAD

**Discussion by vilanovi** [etsy/open-api#1234] [Q&A]
We have a comercial access app and we are experiencing an issue when requesting data for a user with etsy shop id 25463380.

Here a sample CURL of what we do: (note we do not include in the curl our API_KEY nor the ACCESS_TOKEN of the user).

curl -X GET 'https://api.etsy.com/v3/application/shops/25463380/listings?state=active&offset=45&limit=1&includes=Inventory,Images' -H 'Accept: application/json' -H 'Content-Type: application/x-www-form-urlencoded' -H 'x-api-key: API_KEY' -H 'Authorization: Bearer ACCESS_TOKEN’

In this case, when requesting specifically the element number 45 of the list (or this element is included in the page as per example offset = 30, limit = 30), the endpoint returns BAD GATEWAY: {"error":"Bad Gateway”}. Note the endpoint works fine when requesting listings not containing that specific element (for example, offset 46 - limit 30, or offset 0 limit 44)

Maybe data is malformed? but in this case the API shouldn't be failing as it does.

**Comment by valigara** [comment]
we faced similar issue with different listings in multiple etsy shops,
we opened ticket for etsy support **5 month ago**,
provided a lot of failed api calls to them,
3 month of trying to get any update from etsy side - no luck,
result till now - zero response from etsy side.

pretty sad but, etsy api support not helpful at all.

try to mail to developer@etsy.com, maybe you be lucky :)
